tests/main/snap-user-service-socket-activation/task.yaml

```diff
@@ -0,0 +1,46 @@
+summary: Check that sockets for user services are created correctly
+
+systems:
+    # Ubuntu 14.04's systemd doesn't have user@.service
+    - -ubuntu-14.04-*
+    # Amazon Linux 2 gives error "Unit user@12345.service not loaded."
+    - -amazon-linux-2-*
+    # Centos 7 gives error "Unit user@12345.service not loaded."
+    - -centos-7-*
+
+kill-timeout: 5m
+
+prepare: |
+    snap set system experimental.user-daemons=true
+    session-tool --kill-leaked
+
+restore: |
+    session-tool -u test --restore
+    #shellcheck source=tests/lib/user.sh
+    . "$TESTSLIB/user.sh"
+    purge_user_session_data
```

Comment:
Same question as below, what is this removing that is not removed by normal session termination?